test/src/test/java/org/corfudb/integration/ClusterReconfigIT.java

```diff
@@ -117,4 +121,57 @@ public void addNodesTest() throws Exception {
         shutdownCorfuServer(corfuServer_2);
         shutdownCorfuServer(corfuServer_3);
     }
+
+    @Test
+    public void resetTest() throws Exception {
+
+        final int PORT_0 = 9000;
+
+        Process corfuServer = runSinglePersistentServer(corfuSingleNodeHost, PORT_0);
+
+        CorfuRuntime corfuRuntime = createDefaultRuntime();
+        Layout l = new Layout(corfuRuntime.getLayoutView().getLayout());
+        l.setRuntime(corfuRuntime);
+        l.setEpoch(l.getEpoch() + 1);
+        l.moveServersToEpoch();
+        corfuRuntime.getLayoutView().updateLayout(l, 1L);
+        corfuRuntime.invalidateLayout();
+        assertThat(corfuRuntime.getLayoutView().getLayout().getEpoch()).isEqualTo(1L);
+        corfuRuntime.getRouter("localhost:9000").getClient(BaseClient.class).reset().get();
+
+        corfuRuntime = createDefaultRuntime();
+        while (corfuRuntime.getLayoutView().getLayout().getEpoch() != 0L) {
+            Thread.sleep(PARAMETERS.TIMEOUT_SHORT.toMillis());
+            corfuRuntime = createDefaultRuntime();
+        }
+        assertThat(corfuRuntime.getLayoutView().getLayout().getEpoch()).isEqualTo(0L);
+        assertThat(shutdownCorfuServer(corfuServer)).isTrue();
+    }
+
+    @Test
+    public void restartTest() throws Exception {
```

Comment:
comments. 
Also please write tests for cluster scenarios.
3 node cluster with one node reset
3 node cluster with one node restart
Also if you could do code reuse as there is common code in the  2 tests.